src/ansys/fluent/core/quantity.py

```diff
@@ -717,16 +717,98 @@ def __eq__(self, other):
     def __neg__(self):
         return Quantity(-self.value, self.unit)
 
-    def __ge__(self, other):
-        if (
-            isinstance(other, Quantity)
-            and self.get_dimensions_list() == other.get_dimensions_list()
-            and self._si_unit == other._si_unit
+    def __gt__(self, other):
+        if isinstance(other, Quantity) and (
+            self.get_dimensions_list() != other.get_dimensions_list()
         ):
+            raise ValueError("Incompatible dimensions.")
```

Comment:
Future refactor: write a single method that checks that the dimensions are compatible and raises an exception if they are not. The method can accept float arguments which are handled as dimensionless quantities. 
```
# throws
def validate_matching_dimensions(self, other):
```

Reply:
Done. Thank you very much.